src/include/ipc4/copier.h

```diff
@@ -32,6 +32,7 @@
 #include <sof/compiler_attributes.h>
 #include <sof/audio/buffer.h>
 #include <sof/audio/pcm_converter.h>
+#include <sof/audio/component.h>
```

Comment:
yes, it can resolve, just curious is it a standard when can use struct declaration and header file usage is just a structure pointer, then we can try to declare it instead of include a header file?